apps/extension/src/background/keyring/types.ts

```diff
@@ -0,0 +1,32 @@
+export type Bip44Path = {
+  account: number;
+  change: number;
+  index: number;
+};
+
+export type MnemonicState = {
+  id: string;
+  alias?: string;
+  phrase: Uint8Array;
```

Comment:
nit: listing optionals last
```suggestion
  id: string;
  phrase: Uint8Array;
  alias?: string;
```

Reply:
See my comment at https://github.com/anoma/namada-interface/pull/71/#discussion_r988716978 - I think there are different ways of looking at this.